src/app-gocardless/bank-factory.js

```diff
@@ -21,6 +21,7 @@ import SpkMarburgBiedenkopfHeladef1mar from './banks/spk-marburg-biedenkopf-hela
 import SpkKarlsruhekarsde66 from './banks/spk-karlsruhe-karsde66.js';
 import VirginNrnbgb22 from './banks/virgin_nrnbgb22.js';
 import Berliner_Sparkasse_beladebexxx from './banks/berliner_sparkasse_beladebexxx.js';
+import KBCkredbebb from './banks/kbc_kredbebb.js';
```

Comment:
Could you move this to the correct place alpahetically please? Feel free to also move `Berliner_Sparkasse_beladebexxx` to the right place too.

Reply:
Done ✅